vue.config.js

```diff
@@ -11,6 +12,10 @@ const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
 module.exports = {
   configureWebpack: {
     plugins: [
+      // this is needed to remove existing service workers on users' systems
+      // ref: https://www.npmjs.com/package/webpack-remove-serviceworker-plugin
+      // ref: https://github.com/NekR/self-destroying-sw/tree/master/packages/webpack-remove-serviceworker-plugin
+      new RemoveServiceWorkerPlugin(),
```

Comment:
No filename is specified here because the default one is OK. After a `npm run build`, the dist/ folder includes file `sw.js` that is installed by the plugin.